tests/apitest/src/CoreGraphics/CGDisplay.cs

```diff
@@ -0,0 +1,99 @@
+﻿using NUnit.Framework;
+using System;
+
+#if !XAMCORE_2_0
+using MonoMac.AppKit;
+using MonoMac.ObjCRuntime;
+using MonoMac.CoreImage;
+using MonoMac.CoreGraphics;
+using CGSize = System.Drawing.SizeF;
+using CGRect = System.Drawing.RectangleF;
+#else
+using AppKit;
+using ObjCRuntime;
+using CoreImage;
+using CoreGraphics;
+#endif
+
+namespace Xamarin.Mac.Tests
+{
+	[TestFixture]
+	public class CGDisplayTests
+	{
+		int mainDisplayId;
+
+		[SetUp]
+		public void Setup ()
+		{
+			mainDisplayId = CGDisplay.MainDisplayID;
+		}
+
+		[Test]
+		public void CGDisplayShouldGetDimensions ()
+		{
+			var bounds = CGDisplay.GetBounds (mainDisplayId);
+			Assert.IsFalse (bounds.IsNull ());
+			Assert.IsFalse (bounds.IsEmpty);
```

Comment:
When there's more than one `Assert` in a method please always add a (string) message on all of them. Without debug symbols (or if they are broken) it's the only way to be sure which test really failed.